src/main/resources/anniv5Resources/localization/eng/startuppack/Cardstrings.json

```
@@ -0,0 +1,54 @@
+{
+  "${ModID}:MessUp": {
+    "NAME": "Mess Up",
+    "DESCRIPTION": "Deal damage equal to the amount of cards in your draw pile. NL Shuffle your Discard pile into your deck.",
+    "UPGRADE_DESCRIPTION": "Shuffle your Discard pile into your deck. NL Deal damage equal to the amount of cards in your draw pile.",
+    "EXTENDED_DESCRIPTION": [
+      " NL (Deals !D! damage.)"
+    ]
+  },
+  "${ModID}:PlannedDefense": {
+    "NAME": "Planned Defense",
+    "DESCRIPTION": "Innate NL Gain !B! Block. If you played another Innate card this turn, also gain !B! Block next turn"
+  },
+  "${ModID}:SecondStrike": {
+    "NAME": "Second Strike",
+    "DESCRIPTION": "Innate Retain NL Deal !D! damage. If it is not the first turn, play this again"
+  },
+  "${ModID}:DashIn": {
+    "NAME": "Dash In",
+    "DESCRIPTION": "Innate NL Gain !B! Block. NL Deal !D! damage to all enemies X times.",
+    "UPGRADE_DESCRIPTION": "Innate NL Gain !B! Block. NL Deal !D! damage to all enemies X+1 times."
+  },
+  "${ModID}:InstinctiveThinking": {
+    "NAME": "Instinctive Thinking",
+    "DESCRIPTION": "Each turn, if the first card you play is Innate, gain !M! Block."
+  },
+  "${ModID}:Cross": {
+    "NAME": "Cross",
+    "DESCRIPTION": "Deal !D! damage. Deal damage for all your created Crosses. Shuffle a Cross into your draw pile.",
```

```suggestion
    "DESCRIPTION": "Deal !D! damage. Deal damage for all your created *Crosses. Shuffle a *Cross into your draw pile.",
```
Highlights the word yellow without creating a keyword